package.json

```diff
@@ -19,26 +19,27 @@
     "devops"
   ],
   "dependencies": {
-    "expand-template": "^1.0.2",
+    "expand-template": "^1.0.3",
```

Comment:
changes like this will float in automatically and thus this only clutters the history unnecessarily